jsapp/js/editorMixins/editableForm.es6

```diff
@@ -339,7 +339,7 @@ export default assign({
         .catch((resp) => {
           var errorMsg = `${t('Your changes could not be saved, likely because of a lost internet connection.')}&nbsp;${t('Keep this window open and try saving again while using a better connection.')}`;
           if (resp.statusText !== 'error') {
-            errorMsg = resp.statusText;
+            errorMsg = escapeHtml(resp.statusText);
```

Comment:
Do we need to escape this text that is coming from backend?

Reply:
Could take a look. I didn't exercise this error, or try to figure out what the backend really sends here. I escaped it out of caution, because it seemed unlikely that the backend would send formatted text intentionally, and I was concerned that an error text might somehow contain an unescaped user-provided field name.

Reply:
Probably not worth spending the time pursuing the error message. Let's leave it as is :)